firefox-ios/Client/Frontend/Autofill/Address/AddressListViewModel.swift

```diff
@@ -35,7 +35,10 @@ final class AddressListViewModel: ObservableObject, FeatureFlaggable {
 
     private let logger: Logger
 
-    var isEditingFeatureEnabled: Bool { featureFlags.isFeatureEnabled(.addressAutofillEdit, checking: .buildOnly) }
+    var isEditingFeatureEnabled: Bool {
+        AddressLocaleFeatureValidator.isValidRegion() ||
+        featureFlags.isFeatureEnabled(.addressAutofillEdit, checking: .buildOnly)
+    }
```

Comment:
@issammani won't this enable for all even if the reason is valid?

I am not sure if we want the logic to first check feature enabled then validate via region

Reply:
Unless I am missing something `featureFlags.isFeatureEnabled(.addressAutofillEdit, checking: .buildOnly) ` should only be true if the user is enrolled in the experiment ? My reasoning to leave it there was because we want to rollout to germany and france at some point so we will need a nimbus flag again. But I can remove it from this check and keep the flag in the nimbus manifest for later use. What do you think ?

Reply:
Ah sorry, I understand why we need the || with your original change. Please change it back to the way it was before :)